### ISSUE TITLE
don't get notification in distribution .ipa

### ISSUE THREAD

**Issue by sabrina0713** [googlesamples/google-services#182]
Everything works fine in development environment, either 8.0 or 9.0. However, cannot receive notification in .ipa.  Send message through topics. Any advice on how to find out the cause? Also, it might not  related, the token is changed in the .ipa.

**Comment by silvolu** [comment]
What is the value of `kGGLInstanceIDAPNSServerTypeSandboxOption` in the call to `tokenWithAuthorizedEntity`? It should be NO for ad hoc, because as far as I know you cannot use the sandbox environment.

**Comment by sabrina0713** [comment]
You are right, I set it as 'true' (in swift).  However,  the problem still exists

**Comment by silvolu** [comment]
If you set it to true, then you are using the sandbox, which doesn't work for adhoc. Can you set it to false and see it it works?

**Comment by sabrina0713** [comment]
Sorry for not clarifying it. I originally set it to 'true', after  read your response, I changed it to 'false' and it doesn't work.

**Comment by enricsa** [comment]
Same problem here

**Comment by sabrina0713** [comment]
I got the problem solved. Like Silvano Luciani said "changed the SandboxOption". Besides, get a new plist file for PRODUCTION from https://developers.google.com/mobile/add?platform=ios&cntapi=gcm&cntapp=Default%20Demo%20App&cnturl=https:%2F%2Fdevelopers.google.com%2Fcloud-messaging%2Fios%2Fstart%3Fconfigured%3Dtrue&cntlbl=Continue%20with%20Try%20Cloud%20Messaging
And don't forget to use the Authentication token in the 'sending' side

**Comment by enricsa** [comment]
Are there differents plist files development/production?

I have only adapted de gcm example and changed the topic. On debug (development) is working well but not in production. Also i generated again the p12, provisionings, signing identities and changed the sandbox to NO.

**Comment by sabrina0713** [comment]
From what I have experienced, there are different plist files for production and development. I redo all the p12, signing identities to generate that plist file.
 
 
Liping MarlowIT Architect, Global Solution Center214-686-3936lmarlow@us.ibm.com
1177 S Beltline Rd, Coppell, TX 75019
 
 
----- Original message -----From: Enric Sánchez <notifications@github.com>To: googlesamples/google-services <google-services@noreply.github.com>Cc: Liping Marlow/Dallas/IBM@IBMUSSubject: Re: [googlesamples/google-services] don't get notification in distribution .ipa (#182)Date: Tue, Apr 5, 2016 1:53 AM 
Are there differents plist files development/production?
I have only adapted de gcm example and changed the topic. On debug (development) is working well but not in production. Also i generated again the p12, provisionings, signing identities and changed the sandbox to NO.
